src/ua.c

```diff
@@ -533,10 +533,6 @@ static void ua_destructor(void *arg)
 	mem_deref(ua->pub_gruu);
 	mem_deref(ua->acc);
 
-	if (list_isempty(&uag.ual)) {
-		sip_close(uag.sip, false);
-	}
-
```

Comment:
the logic here is that if the last UA is deleted, it should also start the closing
of the SIP stack (call to `sip_close`)...

I guess in your usecase this is probably not wanted ? I guess you want to add and delete
UAs many times ?

Reply:
Yes, my idea is to have my application load UAs on startup and remove them on shutdown. Without removing baresip would just append only :-)